docs/docs/user-guide/getting-started/using-slurm.md

```diff
@@ -0,0 +1,339 @@
+
+#### **Example 1: Running an interactive job**
+
+```bash
+srun --partition=batch --nodes=1 --ntasks=8 --time=00:30:00 --pty bash
+```
+
+This starts an interactive session with 8 tasks on one node for 30 minutes.
+
+#### **Example 2: Running a command on allocated resources**
+
+```bash
+srun --ntasks=4 python train.py
+```
+
+This runs `train.py` using 4 tasks (e.g., GPUs or CPU cores).
+
+### **`squeue`** (Check Job Status)
+
+The `squeue` command displays the status of jobs in the SLURM queue.
+
+#### **Example 1: Viewing all jobs in the queue**
+
+```bash
+squeue
+```
+
+Lists all running and pending jobs.
+
+#### **Example 2: Checking the status of your own jobs**
+
+```bash
+squeue -u $USER
+```
+
+Shows only your jobs in the queue.
+
+#### **Example 3: Checking the status of your team's jobs**
+
+```bash
+squeue -A <account_name>
+```
+
+Shows only your team's jobs in the queue.
+
+### **`sacct`** (Check Job History)
+
+The `sacct` command retrieves historical job information, including resource usage and exit statuses.
+
+#### **Example 1: Viewing completed jobs**
+
+```bash
+sacct
+```
+
+Displays jobs that have finished or failed.
+
+#### **Example 2: Checking details of a specific job**
+
+```bash
+sacct -j 123456 --format=JobID,State,Elapsed,MaxRSS
+```
+
+Shows the **state, runtime, and memory usage** of job `123456`.
+
+### **`scancel`** (Cancel a Job)
+
+The `scancel` command cancels running or pending SLURM jobs.
+
+#### **Example 1: Canceling a specific job**
+
+```bash
+scancel 123456
+```
+
+Cancels job `123456`.
+
+#### **Example 2: Canceling all of your jobs**
+
+```bash
+scancel -u $USER
+```
+
+Cancels all jobs belonging to the current user.
+
+## 2. Example SBATCH script
+
+The following SBATCH script runs a BioNeMo Evo 2 training job inside a containerized environment.
+
+We'll walk through the script in detail below.
+
+```bash
+#!/bin/bash
+# SLURM directives
+# =========================
+#SBATCH --account=[INSERT ACCOUNT HERE]
+#SBATCH --nodes=1
+#SBATCH --partition=[INSERT PARTITIONs HERE]
+#SBATCH --ntasks-per-node=8
+#SBATCH --time=00:15:00
+#SBATCH --mem=0
+#SBATCH --job-name=[INSERT JOB NAME HERE]
+#SBATCH --mail-type=FAIL
+#SBATCH --exclusive
+set -x # Enable debugging (prints executed commands)
+# Job-specific parameters
+# =========================
+IMAGE_NAME=[INSERT IMAGE NAME HERE]
+EXPERIMENT_NAME=[INSERT PROJECT NAME HERE]
+MODEL_SIZE=7b
+CP_SIZE=1
+TP_SIZE=1
+PP_SIZE=1
+MICRO_BATCH_SIZE=2
+GRAD_ACC_BATCHES=1
+SEQ_LEN=8192
+MAX_STEPS=100
+VAL_CHECK=50
+CLIP_GRAD=250
+EXTRA_ARGS="--enable-preemption --use-megatron-comm-overlap-llama3-8k --ckpt-async-save --overlap-grad-reduce --clip-grad $CLIP_GRAD --eod-pad-in-loss-mask"
+EXTRA_ARG_DESC="BF16_perf_cg250_continue"
+LR=0.0003
+MIN_LR=0.00003
+WU_STEPS=2500
+# 0xDEADBEEF
+SEED=1234
+WD=0.1
+ADO=0.01
+HDO=0.01
+# Mounts
+# =========================
+DATA_PATH=/lustre/.../[INSERT DATA PATH HERE]
```

Comment:
this also seems specific to our cluster resources

Reply:
I added a message to the opening paragraph:

> The scripts are configured for the [DGX cloud SLURM environment](https://docs.nvidia.com/dgx-cloud/slurm/latest/cluster-user-guide.html). While some settings are specific to our cluster resources, these scripts can be adapted for other HPC environments.